pkgs/development/tools/clang-tools/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, llvmPackages }:
+{ lib, stdenv, llvmPackages, cxx ? false }:
```

Not sure if this is better?

```suggestion
{ lib, stdenv, llvmPackages, clang ? llvmPackages.clang}:
```

Than you could pass different version of the compiler?

---

I am not sure about this, since llvmPackages is already present